models/general/smalltalk/en/aboutsusi.txt

```diff
@@ -3,3 +3,34 @@ what is your name ? | What's your name ? | Who are you ?| what should i call you
 Hi ! My name is Susi. What's yours?
 Where do you belong? | where are you from | where do you live
 If I tell you, I'll have to kill you | That'll be a secret :D
+
+!example:susi 
+!description: to remember its own name
```

Comment:
The description should actually describe things in several sentences and correct spelling and syntax.